src/ImageSequencer.js

```diff
@@ -77,17 +93,28 @@ ImageSequencer = function ImageSequencer(options) {
     return this;
   }
 
+  /**
+   * @method removeStep
+   * @description Removes the step at the specified index from the sequence.
+   * @param {Object} ref ImageSequencer instance.
+   * @param {Number} index Index of the step to be removed.
+   * @returns {Null}
+   */
   function removeStep(ref, index) {
-    //remove the step from images[image].steps and redraw remaining images
+    // Remove the step from images[image].steps and redraw remaining images.
```

Comment:
Either write always above the function or below the function.

Reply:
That line describes the code inside the function, not the function. There is a JSDoc comment for the function already.